x-pack/solutions/search/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/pipelines.tsx

```diff
@@ -65,6 +65,16 @@ export const SearchIndexPipelines: React.FC = () => {
   const { makeRequest: revertPipeline } = useActions(RevertConnectorPipelineApilogic);
   const apiIndex = isApiIndex(index);
   const extractionDisabled = getContentExtractionDisabled(index);
+  const buttonRef = useRef<HTMLButtonElement | null>(null);
+
+  const onCloseHandler = () => {
+    closeDeleteModal();
+    setTimeout(() => {
+      if (buttonRef.current) {
+        buttonRef.current.focus();
+      }
+    }, 100);
+  };
```

Comment:
We should create this with the `useCallback` hook.
```suggestion
  const onCloseHandler = useCallback(() => {
    closeDeleteModal();
    setTimeout(() => {
      if (buttonRef.current) {
        buttonRef.current.focus();
      }
    }, 100);
  }, [closeDeleteModal, buttonRef]);
```